frontend/src/components/UserSearch.vue

```diff
@@ -187,12 +221,25 @@ export default {
     },
     searchUser: function(queryParameters) {
       this.userSearchLoadingStatus = true;
+      let roleId = this.roleInput.trim();
 
       UsersRepository.get(
         "?briefRepresentation=false&first=0&max=300" + queryParameters
```

Comment:
The 300 limit on search results might be a significant issue for this role search feature. The console currently:

1. Runs a search uses all the query parameters except for role (limit 300).
1. Runs a separate search to get all users that have the specified role (with no limit?).
1. Filters the first search result set using the second.

The problem is that if there are 1000 users with role "GIS - Admin", the search will only ever return 300 at best.